docs/learn-glossary.md

```diff
@@ -90,7 +90,7 @@ Temporal is backed by a multi-tenant service and the unit of isolation is called
 - By default a Temporal service is provisioned with a "default" Namespace. All APIs and tools, such as the UI and CLI, default to the "default" Namespace if it is not specified. So, if you are not planning to use multiple Namespaces, we recommend using the default one.
 - [**Task Queue**](#task-queue) names as well as [**Workflow Ids**](#workflow-id) correspond to a specific Namespace. For example, when a Workflow is started, it is started within a specific Namespace.
 - Temporal guarantees a unique [**Workflow Id**](#workflow-id) within a Namespace, and supports running [**Workflow Executions**](#workflow-execution) to use the same [**Workflow Id**](#workflow-id) if they are in different Namespaces.
-- Various configuration options like the retention period or Archival destination are configured per Namespace as well through a special CRUD API or through [`tctl`](./learn-cli).
+- Various configuration options like the retention period or Archival destination are configured per Namespace as well through a special CRUD API or through [`tctl`](/docs/learn-cli).
```

Comment:
Can also be changed to just [`tctl`](learn-cli) - by linking to the page id - 
Docusaurus docs are not clear about that https://v2.docusaurus.io/docs/markdown-features#referencing-other-documents but you can do that too - just an FYI - as that is how I have been doing it.

Reply:
@cullywakelin initially we used _linking to page id_, however there was/possibly still is a docusaurus bug #5 so we switched to relative paths